pkg/tasks/update_template_content.go

```diff
@@ -84,7 +84,15 @@ func UpdateTemplateContentHandler(ctx context.Context, task *models.TaskInfo, qu
 	if err != nil {
 		return err
 	}
-	return t.RunCandlepin()
+	err = t.RunCandlepin()
+	if err != nil {
+		return err
+	}
+	err = t.daoReg.Template.UpdateLastError(t.ctx, template.OrgID, template.UUID, "")
+	if err != nil {
+		return err
+	}
```

Comment:
I wonder if it would make sense to do this at the start of this task instead of the end, as:

1) if this task runs successfully, it should correct any issue
2) if this task fails, its likely failing in the same way, or at least is the more updated error